docs/baremetal-agent-controller.md

```diff
@@ -262,6 +249,8 @@ Check that the `inspect.metal3.io` and `automatedCleaningMode` are both set to `
 prevent Ironic from doing inspection and any cleaning, which will speed up the deployment process
 and prevent it from running IPA before running the ISO.
 
+This should be set automatically by BMAC, but you never know...
```

Comment:
Maybe write it without expressing uncertainty? What are the actions we can take if it's not set automatically by BMAC?

Reply:
Done, rewritten with a link to the code responsible for setting it